web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/grid.html

```diff
@@ -75,7 +75,12 @@ <h1>
              title="{{'workingCopy' | translate}}"
              class="gn-workingcopy-status pull-right">
           <i class="fa fa-pencil"></i>
-          <span>{{'workingCopy' | translate}}</span>
+          <a href=""
+             data-ng-show="md.isTemplate != 's'"
```

Comment:
@wangf1122 - What is the purpose of having an extra condition?  Could you explain why we would want the pencil icon but no "working copy" text when isTemplate=s?

Reply:
The code was copied from https://github.com/geonetwork/core-geonetwork/blob/fad6dd47e7c6d3377e21bbecd190c562222dd339/web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/editor.html#L86

They perform the same way unless this logic has some issue. So we should fix both places.

Reply:
You may have misunderstood my comments.
The` <div` has `"data-ng-if="(md.draft == 'e') && (md.edit == true)"` which ends making the decision to display the pencil `<i class="fa fa-pencil"></i>`. The condition that you added `data-ng-show="md.isTemplate != 's'"` is only for the text portion.  So if that condition fails the user will only see a pencil (which would be a bug).  Since the pencil is going to be displayed, I believe the text needs to be displayed as well (with or without a link).

i.e. in this case there is a condition for `md.isTemplate != 's'` and `md.isTemplate == 's'`
https://github.com/geonetwork/core-geonetwork/blob/fad6dd47e7c6d3377e21bbecd190c562222dd339/web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/editor.html#L20-L24

As to whether we even need the condition `md.isTemplate != 's'` - I don't know as I don't know what `s` stands for? I'm sure it is in the code somewhere.